get_instagram.py

```diff
@@ -0,0 +1,32 @@
+from InstagramAPI import InstagramAPI
+from key import user_name, password
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+
+    api = InstagramAPI(username=user_name, password=password)
```

Comment:
It's better to move this part into initialization.

Reply:
added